fried-chicken.txt

```diff
@@ -0,0 +1,2 @@
+- cure chicken with salt
+- add egg, buttermilk, hot sauce, pickle juice
```

Comment:
Did you think about adding the temperature of the oil? What type of oil? Temperature of the chicken when it's done?